src/auth/Cargo.toml

```diff
@@ -28,3 +28,7 @@ async-trait = "0.1.83"
 thiserror   = "2"
 http        = "1.2.0"
 time        = "0.3.37"
+
+[dev-dependencies]
+test-case = "3.3.1"
+tokio     = { version = "1.41.1", features = ["macros"] }
```

Comment:
I think we moved to 1.42 elsewhere in the project.  You may also want the `test-util` feature.

Reply:
Updated.

> I think we moved to 1.42 elsewhere in the project.

Everywhere except the `Cargo.toml` I copied. :smile: 

https://github.com/googleapis/google-cloud-rust/blob/e33a1bb86ae33d4c5ef65aa85f88560705b26229/src/gax/Cargo.toml#L47